build_tools/kickstarts/centos-7-adb-vagrant.ks

```diff
@@ -54,6 +54,12 @@ tuned
 sed -i "/HWADDR/d" /etc/sysconfig/network-scripts/ifcfg-eth*
 sed -i "/UUID/d" /etc/sysconfig/network-scripts/ifcfg-eth*
 
+# Add adb version info to consumed by adbinfo
+# https://github.com/projectatomic/adb-atomic-developer-bundle/issues/183
+echo "VARIANT=\"Atomic Developer Bundle (ADB)\"" >> /etc/os-release
+echo "VARIANT_ID=\"adb\"" >> /etc/os-release
+echo "VARIANT_VERSION=\"2.0\"" >> /etc/os-release
```

Comment:
@praveenkumar I think the version should be 1.6.0. As Next release would be 1.6.0